liteclient/client.go

```diff
@@ -108,14 +137,19 @@ func PrepareTransaction(inputsBody string, outputsBody string) string {
 		newTransaction.PushOutput(addr, out.Coins, out.Hours)
 	}
 
-	newTransaction.SignInputs(keys)
+	if signatureList == nil {
```

Comment:
Same `len(signatureList) == 0` comment, generally that is better for checking arrays in go, rather than relying on nil

Reply:
Changed on both places